src/einsteinpy/utils/schwarzschild_utils.py

```diff
@@ -21,12 +20,33 @@ def schwarzschild_radius(mass):
         Schwarzschild radius for a given mass
 
     """
+    if not isinstance(mass, u.quantity.Quantity):
+        mass = mass * u.kg
+    if not isinstance(c, u.quantity.Quantity):
+        c = c * u.m / u.s
+    if not isinstance(G, u.quantity.Quantity):
+        G = G * constant.G.unit
     M = mass.to(u.kg)
-    num = 2 * constant.G * M
-    deno = constant.c ** 2
+    num = 2 * G * M
+    deno = c ** 2
     return num / deno
 
 
+def schwarzschild_radius_dimensionless(M, c=constant.c.value, G=constant.G.value):
+    """
+    Parameters
+    ----------
```

Comment:
Change the docstring.
```
Parameters
------------
mass : float
    Mass of massive body
c : float
    Speed of light, defaults to value of speed of light in SI units.
G : float
    Gravitational Constant, defaults to its value in SI units
```

Reply:
i dont understand i did isort and black but it is still showing failed.